packages/storybook/src/decorators/withFluentProvider.tsx

```diff
@@ -0,0 +1,21 @@
+import { makeDecorator } from '@storybook/addons';
+import { FluentProvider } from '@fluentui/react-provider';
+import { Theme } from '@fluentui/theme';
+import * as React from 'react';
+
+import { useFluentTheme } from '../knobs/useFluentTheme';
+
+const ProviderWrapper: React.FunctionComponent<{ theme?: Theme | undefined }> = props => {
+  const { theme } = useFluentTheme();
+
+  return <FluentProvider theme={theme}>{props.children}</FluentProvider>;
+};
+
+export const withFluentProvider = makeDecorator({
```

Comment:
will you add this to `react-examples` in a separate PR ?

Reply:
It's done per package, for `react-avatar` it's done in #16793.

Reply:
I see the author has to explicitly enable this decorator